### PR TITLE
Unit test d - input

### PR DESCRIPTION
**Pull request by KristerJazz**

### CODE REVIEW

File: tests/unit-test-d-input/unit-test-d-input.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+
+Unit test - Input function
+Contributor - Krister Jazz Urog
+
+*/
+#include "unit-test-d-input.h"
```
Comment (lleon95) [inline tests/unit-test-d-input/unit-test-d-input.c]: This should be imported last

File: tests/unit-test-d-input/unit-test-d-input.c [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+int test_equality(double in, double test_value) {
+  if (in == test_value)
+    return 1;
+  else
+    return 0;
+}
+
+int main(int argc, char **argv) {
+  char restfile[BLEN], trajfile[BLEN], ergfile[BLEN], line[BLEN];
+  // int testing_progress=0;
```
Comment (lleon95) [inline tests/unit-test-d-input/unit-test-d-input.c]: Get rid of useless comments

File: tests/unit-test-d-input/unit-test-d-input.h [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/* Sample input values */
+
+const int NATOMS = 108;               //natoms
+const double MASS = 39.948;           // mass in AMU
+const double EPSILON = 0.2379;        // epsilon in kcal/mol
+const double SIGMA = 3.405;           // sigma in angstrom
+const double RCUT = 8.5;              // rcut in angstrom
+const double BOX = 17.1580;           // box length (in angstrom)
+//char = argon_108.rest;              // restart
```
Comment (lleon95) [inline tests/unit-test-d-input/unit-test-d-input.h:10]: Remove useless comments

File: tests/unit-test-d-input/unit-test-d-input.h [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/* Sample input values */
```
Comment (lleon95) [inline tests/unit-test-d-input/unit-test-d-input.h]: Put header guards

**Review by lleon95 — CHANGES_REQUESTED** [review]

**Review by stefanocampanella — APPROVED** [review]

**Review by lleon95 — APPROVED** [review]

**Pull request merged**